Makefile.include

```diff
@@ -106,6 +106,11 @@ OS := $(shell uname)
 # set python path, e.g. for tests
 PYTHONPATH := $(RIOTBASE)/dist/pythonlibs/:$(PYTHONPATH)
 
+# set default PROGRAMMER value if flashing a board on IoT-LAB
+ifneq (,$(IOTLAB_NODE))
+  PROGRAMMER = iotlab
```

Comment:
Keep it with `?=` to allow easily overwriting it from the environment in case it is required.

Reply:
Done and comment added.